gateway/1.4/modules/ROOT/pages/_partials/task-reg-run-flex-gateway.adoc

```diff
@@ -74,6 +74,36 @@ to generate set of instructions that includes a command block with the registrat
 // tag::token-k8s-heading[]
 == Register and Run with a Token in a Kubernetes Cluster
 // end::token-k8s-heading[]
+// tag::app-openshift-heading[]
+== Register and Run with a Connected App in an OpenShift Cluster
+// end::app-openshift-heading[]
+// tag::user-openshift-heading[]
+== Register and Run with a Username and Password in an OpenShift Cluster
+// end::user-openshift-heading[]
+// tag::token-openshift-heading[]
+== Register and Run with a Token in an OpenShift Cluster
+// end::token-openshift-heading[]
+
+// tag::note-openshift-k8[]
+From the command line, OpenShift procedures match Kubernetes procedures. 
+// end::note-openshift-k8[]
+
+/ logos and links to sections that _use anchors_ in install and reg/run pages
```

Comment:
single "/" correct here?

Reply:
If comment, do you need the emphasis?

Reply:
Yes, I want to emphasize it so people reading the comments are sure to notice it.

Reply:
Does the single "/" work as a comment?

Reply:
I fixed it, but since this is in a partial, and we're using tags, it never caused a problem.